lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -0,0 +1,20 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyIndexer
+  class Configuration
+    extend T::Sig
+
+    sig { returns(T::Array[String]) }
+    attr_accessor :files_to_index
+
+    sig { void }
+    def initialize
+      files_to_index = $LOAD_PATH.flat_map { |p| Dir.glob("#{p}/**/*.rb", base: p) }
+      files_to_index.concat(Dir.glob("#{Dir.pwd}/**/*.rb"))
+      files_to_index.reject! { |path| path.end_with?("_test.rb") }
```

Comment:
Same, it seems it should be configurable.

Maybe you can take some inspiration from https://github.com/Shopify/spoom/blob/main/lib/spoom/file_collector.rb.

Reply:
It is configurable. You can invoke `RubyIndexer.configure` and change it. But again, this PR is only moving the existing code inside, the next ones will tune this.